spec/policies/team_policy_spec.rb

```diff
@@ -31,8 +31,11 @@
   end
 
   describe 'show?' do
-    it 'is always true' do
-      expect(subject.show?).to eq(true)
+    before do
+      allow(subject).to receive(:is_editable_by?).with(user.id).and_return('is editable by result')
+    end
+    it 'is the value of is_editable_by? for the user_id' do
+      expect(subject.edit?).to eq(subject.send(:is_editable_by?, user.id))
```

Comment:
I don't think this is clear what you are testing here.  I think it is that the `show?` method calls `#is_editable_by?` and returns the result of that call - is that right?

If so, I think something on these lines makes more sense:
```
expect(subject).to receive(:is_editable_by?).with(user_id).and_return(:result_from_is_editable_by?)
expect(subject.edit?).to return(:result_from_is_editable_by?)
```

I'm assuming that the private method `#is_editable_by?` is being tested elsewhere.